pkg/kubefed2/federate/federate.go

```diff
@@ -43,10 +43,11 @@ var (
 	federate_long = `
 		Federate creates a federated resource from a kubernetes resource.
 		The target resource must exist in the cluster hosting the federation
-		control plane. The control plane must have a FederatedTypeConfig
-		for the type of the kubernetes resource. The new federated resource
-		will be created with the same name and namespace (if namespaced) as
-		the kubernetes resource.
+		control plane. If the federated resource needs to be created in the
+		API, the control plane must have a FederatedTypeConfig for the type
+		of the kubernetes resource. If using with flag '-o yaml', it is not
+		necessary for the FederatedTypeConfig to exist. Infact federate can
```

Comment:
s/exist. Infact.*/exist (or even for the federation API to be installed in the cluster)./

Reply:
Updated!